serialize.go

```diff
@@ -97,7 +93,7 @@ func RawSerialize(tn string, b interface{}) ([]Field, error) {
 		if kindOfField := field.Type.Kind(); kindOfField == reflect.Struct {
 			if fieldConverted := fieldValue.Convert(fieldValue.Type()); fieldConverted.CanInterface() {
 				payload := fieldConverted.Interface()
-				serialized, err := RawSerialize(tn, payload)
+				serialized, err := serialize(payload, tn...)
```

Comment:
Pass more than only one tag isn't a interesting feature for the project at moment.